pypeclub/OpenPype
# TrayPublisher: implemented render_mov_batch

## Brief description
Re-implementation of `render_mov_batch` family for TrayPublisher.

## Description
This family is used to publish multiple .mov files at once. Artists doesn't need to select context, some heuristic is implemented to parse asset name from a file name (`chair.mov`, `chair_v001.mov`...).
Resulting task name is configured in Settings. (Could be configured multiple ['Compositing', 'Art'] - first available task on Asset will be picked up.


## Testing notes:
1. Upload single or multiple .mov files in TrayPublisher.
2. Settings default contains 'Compositing' as a resulting task name
3. Try to publish .mov file with dummy name, it should warn you, that file name doesn't contain valid asset name
